### PR TITLE
Add navigation launchers

### PR DESCRIPTION
**Pull request by jginesclavero**
I have included the required launchers to navigate into the Among-us world:
Screenshot from 2020-10-13 13-09-21

- 2 launchers, each one for one colour.
- The map of the scenario.
- Rviz tools to control the robot independently. 
image

#Hacktober

### CODE REVIEW

**Review by P4B5 — APPROVED** [review]

File: navigation/launch/includes/amcl.launch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <arg name="initial_pose_x"  default="0.0"/>
+  <arg name="initial_pose_y"  default="0.0"/>
+  <arg name="initial_pose_a"  default="0.0"/>
+  <arg name="odom_frame_id"   default="odom"/>
+  <arg name="base_frame_id"   default="base_footprint"/>
+  <arg name="global_frame_id" default="map"/>
+  <arg name="tf_prefix" default="tf_p"/>
+
+  <node pkg="amcl" type="amcl" name="amcl">
+  	<param name="tf_broadcast" value="true"/>
```
Comment (fgonzalezr1998) [inline navigation/launch/includes/amcl.launch]: here, the tab is not good

File: navigation/launch/navigation.launch [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <arg name="open_rviz" default="true"/>
+  <arg name="move_forward_only" default="false"/>
+  <arg name="tf_prefix" default=""/>
+  <arg name="initial_pose_x" default="0"/>
+  <arg name="initial_pose_y" default="0"/>
+  <arg name="initial_pose_a" default="0"/>
+  <!-- AMCL -->
+  <include file="$(find navigation)/launch/includes/amcl.launch">
+    <arg name="tf_prefix" value="$(arg tf_prefix)"/>
+		<arg name="scan_topic" value="scan" />
```
Comment (fgonzalezr1998) [inline navigation/launch/navigation.launch]: Bad tabulation

File: navigation/launch/purple_tb3_navigation.launch [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  <node name="map_server" pkg="map_server" type="map_server" args="$(find navigation)/maps/map.yaml" respawn="false">
+    <param name="frame_id" value="map" />
+  </node>
+  
+  <group ns="$(arg tf_prefix)">
+    <param name="tf_prefix" value="$(arg tf_prefix)" />
+    <include file="$(find navigation)/launch/navigation.launch">
+      <arg name="tf_prefix" value="$(arg tf_prefix)"/>
+      <arg name="initial_pose_x" value="-3.0" />
+			<arg name="initial_pose_y" value="-1.0" />
```
Comment (fgonzalezr1998) [inline navigation/launch/purple_tb3_navigation.launch]: Bad tabulation

File: navigation/launch/red_tb3_navigation.launch [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  <node name="map_server" pkg="map_server" type="map_server" args="$(find navigation)/maps/map.yaml" respawn="false">
+    <param name="frame_id" value="map" />
+  </node>
+
+  <group ns="$(arg tf_prefix)">
+    <param name="tf_prefix" value="$(arg tf_prefix)" />
+    <include file="$(find navigation)/launch/navigation.launch">
+      <arg name="tf_prefix" value="$(arg tf_prefix)"/>
+      <arg name="initial_pose_x" value="-6.0" />
+			<arg name="initial_pose_y" value="-1.0" />
```
Comment (fgonzalezr1998) [inline navigation/launch/red_tb3_navigation.launch]: Check the tabs

**Comment by jginesclavero** [comment]
@fgonzalezr1998 tabs fixed! thank you

**Pull request merged**